api/routes/petition.py

```diff
@@ -38,12 +38,37 @@ def create_petition(
 
 @router.get("/", response_model=List[PetitionInfo])
 def get_all_petitions(
-    offset: int, limit: int, db: Session = default_session
+    req: Optional[ResponseModificators], db: Session = default_session
```

Comment:
when you add pydantic model here, FastAPI thinks these are body parameters (you can check this in swagger). And `get` requests don't have the body. It could work locally, but it's not up to standard.

Reply:
I fixed it in the last commit